kernel/syscall.c

```diff
@@ -2656,6 +2656,86 @@ long myst_syscall_recvmsg(int sockfd, struct msghdr* msg, int flags)
     return ret;
 }
 
+long myst_syscall_sendmmsg(
+    int sockfd,
+    struct mmsghdr* msgvec,
+    unsigned int vlen,
+    int flags)
+{
+    long ret = 0;
+    myst_fdtable_t* fdtable = myst_fdtable_current();
+    myst_sockdev_t* sd;
+    myst_sock_t* sock;
+    unsigned int cnt;
+
+    ECHECK(myst_fdtable_get_sock(fdtable, sockfd, &sd, &sock));
+    if (!msgvec && vlen)
```

Comment:
Check the parameters before doing actual work, e.g., myst_fdtable_get_sock

Reply:
Fixed.